streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignorTest.java

```diff
@@ -81,7 +81,7 @@
     );
 
     @Test
-    public void shouldBeStickyForActiveAndStandbyTasksEvenIfNoWarmups() {
+    public void shouldBeStickyForActiveAndStandbyTasksWhileWarmingUp() {
```

Comment:
The diff is misaligned. I removed `shouldBeStickyForActiveAndStandbyTasksEvenIfNoWarmups` and added `shouldSkipWarmupsWhenAcceptableLagIsMax`.